interfaces/mount/entry.go

```diff
@@ -67,22 +67,20 @@ func (a *Entry) Equal(b *Entry) bool {
 }
 
 // escape replaces whitespace characters so that getmntent can parse it correctly.
-//
-// According to the manual page, the following characters need to be escaped.
-//  space     => (\040)
-//  tab       => (\011)
-//  newline   => (\012)
-//  backslash => (\134)
-var escape = strings.NewReplacer(" ", `\040`, "\t", `\011`, "\n", `\012`, "\\", `\134`).Replace
+var escape = strings.NewReplacer(
+	" ", `\040`, //   space     => (\040)
+	"\t", `\011`, //  tab       => (\011)
+	"\n", `\012`, //  newline   => (\012)
+	"\\", `\134`, //  backslash => (\134)
```

Comment:
The suffix comments are unnecessary. The code is saying exactly the same thing, and more clearly even.